docs/usage.md

```diff
@@ -568,22 +671,32 @@ If you prefer, you can specify the full path to your reference genome when you r
 --intervals '[path to the intervals file]'
 ```
 
-### --knownIndels
+### --known_indels
 
 If you prefer, you can specify the full path to your reference genome when you run the pipeline:
 
 ```bash
---knownIndels '[path to the knownIndels file]'
+--known_indels '[path to the knownIndels file]'
 ```
 
-### --knownIndelsIndex
+### --knownIndels
+
+> :warning: This params has been removed.
+> Please check: [`--known_indels`](#--known_indels)
+
+### --known_indels_index
 
 If you prefer, you can specify the full path to your reference genome when you run the pipeline:
 
 ```bash
---knownIndelsIndex '[path to the knownIndels index]'
+--known_indels_index '[path to the knownIndels index]'
 ```
 
+### --knownIndelsIndex
+
+> :warning: This params has been removed.
+> Please check: [`--known_indels_index`](#--known_indels_index)
+
 ### --pon
```

Comment:
It might need to be fixed here, that I saw in `main.nf` that now it still filters the variants, even if there is no PON (see comment there)

Reply:
Oh I see, and yes indeed, we added that back.